docs/release-notes/platform_10-9-0_releasenotes.md

```diff
@@ -0,0 +1,124 @@
+
+![Default Providers Selection](./img/v10.9_defaultProvider.png)
+
+This will enhance the governance for every Company Owner for company's projects and give better control over the management of company tools.
+The provider for each capability (Git Provider, CI/CD and Secret Manager) can always be changed in any moment via CMS.
+
+For more information, [go to the documentation](/development_suite/set-up-infrastructure/configure-provider.mdx).
+
+#### GitHub Actions as CI/CD tool in Console
+
+Starting with this version, the console supports GitHub Actions as a CI/CD tool.
+
+Within the Providers section accessible through the Company Overview area you can now configure a GitHub provider as a CI/CD tool and using it for the deployment of your workload.
+
+For more information, [go to the documentation](/development_suite/deploy/.mdx).
+
+#### Managing Service Accounts at Console level
+
+It is possible to create Company-independent service accounts in case you need to give them privileges to perform [root-level operations](/development_suite/identity-and-access-management/console-levels-and-permission-management.md).
+
+:::info
+Only Console Super Users can add or delete Company independent service accounts, manageable exclusively at back-office level.
+:::
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* We fixed an issue in the creation of a template/example that prevented the Git repository owner name from being filled in with special characters.
+
+## Fast Data
+
+### New Features
+
+The file picker of the DDL modal import now supports `.sql` file 
+
+### Improvements
+
+The Console automatic aggregation generation now represents one-to-one relationship configuration as an object.
+
+### Breaking Changes
+
+#### Revamped Single View Trigger Generator
+We're thrilled to announce the release of v.2.0.1 of the Single View Trigger Generator, one of the Fast Data component. In this release, we've given the component a complete overhaul to optimize its performance and streamline its functionality.
+
+However, as with any major update, there are a few breaking changes to be aware of. We've renamed the following environment variables to better clarify their purpose:
+- `STRATEGIES_FOLDER` → `MANUAL_STRATEGIES_FOLDER`
+- `CUSTOM_FUNCTIONS_FOLDER` → `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`.
+We've also removed the `USE_AUTOMATIC` variable since the feature is now enabled in all the service configurations.  
+Furthermore, the following environment variables are now required:
+- `SINGLE_VIEW_NAME`
+- `ER_SCHEMA_FOLDER`
+- `PROJECTION_CHANGES_SCHEMA_FOLDER`
+
+For more details on the service and its behavior, please check out the official [documentation](/fast_data/single_view_trigger_generator.md)
```

Comment:
```suggestion
#### Revamped Single View Trigger Generator

We're thrilled to announce the release of v.2.0.1 of the Single View Trigger Generator, one of the Fast Data component. In this release, we've given the component a complete overhaul to optimize its performance and streamline its functionality.

However, as with any major update, there are a few breaking changes to be aware of. We've renamed the following environment variables to better clarify their purpose:
- `STRATEGIES_FOLDER` → `MANUAL_STRATEGIES_FOLDER`
- `CUSTOM_FUNCTIONS_FOLDER` → `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`.
We've also removed the `USE_AUTOMATIC` variable since the feature is now enabled in all the service configurations.  
Furthermore, the following environment variables are now required:
- `SINGLE_VIEW_NAME`
- `ER_SCHEMA_FOLDER`
- `PROJECTION_CHANGES_SCHEMA_FOLDER`

For more details on the service and its behavior, please check out the official [documentation](/fast_data/single_view_trigger_generator.md).
```